### PR TITLE
Feature/basic validation and standardization test

### PR DESCRIPTION
**Pull request by carlos-verdes**
### Description

IT with validation and standardization + dsl for parsing

### Test
Tests: succeeded 5, failed 0, canceled 0, ignored 0, pending 0

### Coverage
All done. Coverage was [100.00%]

### CODE REVIEW

File: src/main/scala/com/digitalknowledge/etl/spark/package.scala [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        header: Boolean = true,
+        dateFormat: String = "yyyy-MM-dd"): DataFrame =
+      sqlContext.read.format("com.databricks.spark.csv")
+        .option("header", header.toString)
+        .option("inferSchema", inferSchema.toString)
+        .option("dateFormat", "yyyy-MM-dd")
+        .load(file)
+  }
+
+  implicit def rowParsingErrors(self: RDD[Either[Row, ParsingError]]): ParsedItemsUnwrapFunctions[Row] =
```
Comment (jjlopezm) [inline src/main/scala/com/digitalknowledge/etl/spark/package.scala]: I like this structure to control the errors!

Comment (carlos-verdes) [inline src/main/scala/com/digitalknowledge/etl/spark/package.scala]: Good point, if a row has more than one validation error we should inform of both, I will add this scenario on the test.

My question now is if we should retrieve each field error as a different "entry" or only one entry with all the errors together.
Example 1:
```scala
(row1, error1)
(row1, error 2)
```

Example 2:
```scala
(row1, Seq(error1, error2))
```

Comment (jjlopezm) [inline src/main/scala/com/digitalknowledge/etl/spark/package.scala]: My idea is the second example.

Comment (carlos-verdes) [inline src/main/scala/com/digitalknowledge/etl/spark/package.scala]: @Neurus1970 ... algún comentario de algún DDR al respecto?

**Comment by carlos-verdes** [comment]
I updated the code with your suggestions and also refactor some names to leave the code more clear.
Looking forward for your coments!!

**Comment by Neurus1970** [comment]
I've uploaded all available test data to 

https://drive.google.com/a/digital-knowledge.es/file/d/0BwkVy5nzZh0GZkJHS295YzFNd1E/view?usp=sharing

I have also created a new pull request to link this repository with Jenkins.

**Pull request merged**